comps/embeddings/langchain/embedding_tei.py

```diff
@@ -4,7 +4,7 @@
 import os
 import time
 
-from langchain_community.embeddings import HuggingFaceHubEmbeddings
+from langchain_huggingface import HuggingFaceEndpointEmbeddings
```

Comment:
I see a lot of other components using "HuggingFaceHubEmbeddings" instead of "HuggingFaceEndpointEmbeddings". What is the motivation behind this change?

Reply:
Huggingface is moving their langchain integrations into their partner package "langchain_huggingface" rather than the community library. Those in community are deprecated starting in langchain 0.3 https://github.com/langchain-ai/langchain/blob/master/libs/community/langchain_community/embeddings/huggingface_hub.py

Reply:
I see, Rita, good to know :)
@ftian1 @lvliang-intel @lkk12014402 @XuhuiRen , would like to know if you are planning to update the Embedding packages here. According to Rita, it makes sense to do massive replace all "HuggingFaceHubEmbeddings" to "HuggingFaceEndpointEmbeddings"

Reply:
yes, it should be the direction.